2022 - LA - Sons of Horus.cat

```diff
@@ -6358,4 +6358,34 @@ A Warlord with this Trait gains a bonus of +1 to its Leadership Characteristic w
   <catalogueLinks>
     <catalogueLink id="2f60-18fb-e6ed-8574" name="(HH V2) Legions Astartes" targetId="6393-649c-7213-a327" type="catalogue" importRootEntries="true"/>
   </catalogueLinks>
+  <selectionEntries>
+    <selectionEntry id="3b55-a060-d00a-20e7" name="Rite of War" hidden="false" collective="false" import="false" type="upgrade">
+      <constraints>
+        <constraint field="selections" scope="parent" value="1" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="eb90-9235-aa1-ac41" type="min"/>
+        <constraint field="selections" scope="parent" value="1" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="c6fd-e5cb-7f2d-154a" type="max"/>
+      </constraints>
+      <categoryLinks>
+        <categoryLink id="7254-ed68-3002-6891" name="New CategoryLink" hidden="false" targetId="d494-e450-d4aa-579a" primary="true"/>
+      </categoryLinks>
+      <entryLinks>
+        <entryLink id="ae0-abdc-31f7-210b" name="Rites of War (Text Only / Work in Progress)" hidden="false" collective="false" import="true" targetId="cf3e-1d75-6f91-651f" type="selectionEntryGroup">
```

Comment:
Why is the category "Text Only / Work in Progress" ?

Reply:
> Why is the category "Text Only / Work in Progress" ?

That's an odd revert...
I think it's dating back from when the files were initially constructed. So NR has ported over the link with the text for the link as it used to be read when the entry link used to be links to the group selection entry which had that name...